site/content/post/2018/2018-03-28-5-ways-loop-over-DOM-elements-queryselectorall-javascript.md

```diff
@@ -0,0 +1,156 @@
+---
+title: 5 ways to loop over DOM elements from querySelectorAll in JavaScript
+author: Raymon Schouwenaar
+type: post
+date: 2018-03-28T21:15:08+00:00
+url: /5-ways-loop-over-DOM-elements-queryselectorall-javascript/
+featured_image: loop-over-nodelist-queryselectorall-javascript.png
+draft: false
+categories:
+  - Javascript
```

Comment:
__[Goodcheck]__

Misspelling?

GitHub is not Github.
JavaScript is not Javascript.
Facebook is not facebook!


<!-- Comment by [SideCI](https://sideci.com) -->